vpc/service/ec2wrapper/ec2_session.go

```diff
@@ -240,6 +240,7 @@ func (s *EC2Session) UnassignIpv6Addresses(ctx context.Context, unassignIpv6Addr
 	ec2client := ec2.New(s.Session)
 	unassignPrivateIPAddressesOutput, err := ec2client.UnassignIpv6AddressesWithContext(ctx, &unassignIpv6AddressesInput)
 	if err != nil {
+		_ = HandleEC2Error(err, span)
 		return nil, err
```

Comment:
Why does this one not have

    status.Convert(err).Err()

? It looks like it prettifies grpc errors some how?